chroma_core/migrations/0017_sfaenclosure_sfastoragesystem.py

```diff
@@ -47,6 +47,7 @@ class Migration(migrations.Migration):
                 ("element_name", models.TextField()),
                 ("health_state", models.PositiveSmallIntegerField()),
                 ("health_state_reason", models.TextField()),
+                ("model", models.TextField()),
```

Comment:
It is ok to modify the migrations before it hits production, right?

Reply:
yes.